server/service/scenarios/slides/endpoints.js

```diff
@@ -20,7 +20,10 @@ exports.addSlide = asyncMiddleware(async (req, res) => {
 
 exports.orderSlides = asyncMiddleware(async (req, res) => {
     const { id: scenario_id } = reqScenario(req);
-    const slide_ids = req.body.slides;
+    const slide_ids = req.body.slides.map(id => {
+        if (typeof id === 'object') return id.id;
```

Comment:
When would id be an object? Out of curiosity, non-blocking

Reply:
lets you submit `{"slides":[{}, {},{}]}` with ids in the objects, or just the ids of the slides